### DISCUSSION TITLE
Welcome to 일조 Discussions!

### DISCUSSION THREAD

**Discussion by OoooEuna** [OoooEuna/MSA5_TEAM1#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tellmebaby** [comment]
👮‍♂️ 빠르니~ 
![simi](https://github.com/OoooEuna/MSA5_TEAM1/assets/69560522/40a214f8-42c2-4198-87ec-238bb1a50074)

**Comment by Joono0513** [comment]
![images](https://github.com/OoooEuna/MSA5_TEAM1/assets/18590494/4983d167-f8d1-44b9-bb44-e321cf1b632a)

**Reply by tellmebaby** [reply]
커엽네요
